### PR TITLE
Changes are being live upstream to @jjpaulo2 (I hope he likes it!)

### PR DESCRIPTION
**Pull request by yoyomonem**
@jjpaulo2 will now see my changes while I shared the changes to him! Maintainers can't edit this forever. A lot of work has been made. I hope he likes it!

### CODE REVIEW

**Comment by yoyomonem** [comment]
@jjpaulo2! My changes are live upstream in this fork! I hope you like it! ;)

**Comment by jjpaulo2** [comment]
Thanks for your contibuting! 😄 
I'm going to read and test the features.

**Comment by yoyomonem** [comment]
Did you like it?

File: sonic/game/cenarios/GreenHillZone.py [hunk 1]
```diff
@@ -37,9 +41,13 @@ def atualizar_eventos(self, evento):
                 self.sonic.stop()
                 
     def atualizar_cenario(self):
-        if self.sonic.rect[0] > self.tela.get_rect()[2] :
+        if self.sonic.rect[0] >= self.tela.get_rect()[2] :
             pygame.mixer.music.fadeout(500)
-            self.rodando = False
+            self.rodando = 1
```
Comment (jjpaulo2) [inline sonic/game/cenarios/GreenHillZone.py:46]: Are you using `0 = True` and `1 = False`?

**Comment by jjpaulo2** [comment]
I have read your code and tested the project working. I loved what you did. Thanks very much for this contributing. To close and merge, I just need you do somethings:

- Update the `Pipfile` with the `PySimpleGUI` dependency, or write it into a `requirements.txt`
- Remove the code blocks commented from your code

All things else looks good for me.

**Comment by jjpaulo2** [comment]
I closed this PR because of non answers on comments and code reviews.

**Pull request closed without merging**